src/model/primitives/uri.json

```diff
@@ -0,0 +1,3 @@
+{
+    "type": "string"
```

Comment:
We could add a regexp to ensure that it a valid uri.

Reply:
That's a great suggestions. I will work on it and re-submit the PR.

Reply:
@fox1t this has now been resolved.